stdlib/collections/__init__.pyi

```diff
@@ -261,19 +261,21 @@ class defaultdict(Dict[_KT, _VT], Generic[_KT, _VT]):
     @overload
     def __init__(self, **kwargs: _VT) -> None: ...
     @overload
-    def __init__(self, default_factory: Callable[[], _VT] | None) -> None: ...
+    def __init__(self, __default_factory: Callable[[], _VT] | None) -> None: ...
     @overload
-    def __init__(self, default_factory: Callable[[], _VT] | None, **kwargs: _VT) -> None: ...
+    def __init__(self, __default_factory: Callable[[], _VT] | None, **kwargs: _VT) -> None: ...
     @overload
-    def __init__(self, default_factory: Callable[[], _VT] | None, map: Mapping[_KT, _VT]) -> None: ...
+    def __init__(self, __default_factory: Callable[[], _VT] | None, __map: Mapping[_KT, _VT]) -> None: ...
     @overload
-    def __init__(self, default_factory: Callable[[], _VT] | None, map: Mapping[_KT, _VT], **kwargs: _VT) -> None: ...
+    def __init__(self, __default_factory: Callable[[], _VT] | None, __map: Mapping[_KT, _VT], **kwargs: _VT) -> None: ...
     @overload
-    def __init__(self, default_factory: Callable[[], _VT] | None, iterable: Iterable[tuple[_KT, _VT]]) -> None: ...
+    def __init__(self, __default_factory: Callable[[], _VT] | None, __iterable: Iterable[tuple[_KT, _VT]]) -> None: ...
     @overload
-    def __init__(self, default_factory: Callable[[], _VT] | None, iterable: Iterable[tuple[_KT, _VT]], **kwargs: _VT) -> None: ...
-    def __missing__(self, key: _KT) -> _VT: ...
-    # TODO __reversed__
```

Comment:
I removed this comment as `__reversed__` is inherited from `dict`; mypy seems to be able to infer the type fine.